### ISSUE TITLE
Why have you blocked me in twitter?

### ISSUE THREAD

**Issue by ashumeow** [passy/ama#34]
Main question is:-
"Have I done any harm that made you block me??" 
"Have I ever been offensive??"
If you felt that i am "whatever popped into your mind", then you could have probably ignored me or muted me, instead of blocking me.
Just wanted to say honestly that i hated you for few months after you blocked me for no serious reasons. Then, i got actually fed-up and blocked u. Now, i unblocked you because hating is bad.

To be honest,
I generally come into your profile and read your tweets only for knowledge purpose. I like information-sharing or geeky humans because i have never seen such human in real.

First time, i came to know you while i was stacking some repositories in github on 2013 when i was a newbie to github. I found you so cool among one of the developers while going through some of your stuffs. And that's why i read your tweets because i found your stuffs like a Bible. But, your act of blocking really affected me just because of one tweet i just posted you. That day i replied your tweet because i found your tweet interesting which made me wonder why was the tweet not extended with curiosity, so i tweeted you in a questioning way. Just to make clear, i never any intention to annoy you at all. Because, i generally don't communicate with humans at all and not even with my own busy parents. I communicate only with my books, music, laptop, web browser and text editor and more inside my room and then, break stuffs. The reason i use social media was only to myself feel human where i see many humans wandering and sharing information and updates. I do read, write & code for hours i can't remember, then eat when hungry & sleep less. I'm saying all this things because i really never had any specific interest to communicate with u. All i had for you was respect not just seeing your reputation, but by seeing your hardwork and efforts.
You know what? You are not the only person who blocked me. Anyways, I'm writing such huge explanation to you. Thank you.

**Comment by passy** [comment]
Well that's a genuinely interesting question.

I'm gonna be frank, I have absolutely no idea why I originally blocked you, @ashumeow. I use block buttons very generously on the internet. On GitHub, on Facebook, on Twitter and for emails. I just checked, and I'm blocking about 500 people on Twitter at the moment.

It doesn't have to be a really offensive comment, it's enough if someone is truly unhelpful towards one of my friends or sides with a group of people I don't want to hear from (Gamer Gaters on Twitter are one example of that). Obviously, it's very easy to make errors when following those very loose rules and it seems like I made one in your case. I'm sorry about that and I've unblocked you. It isn't my intention to make anyone feel bad for blocking them. Honestly, I'm pretty sure that for 99% of the people I block they could give less of a cr*p that I blocked them.

Despite these problems with my approach to blocking, it's the only way that works for me. Without generously blocking people, I would most certainly already have quit the internet altogether. There's too much negativity, too many unhelpful and demanding people out there and I do a terrible job at not letting myself getting dragged down by those few bad apples.

Thanks for asking that question, I'm sure this wasn't easy to write.
